Scripts/Runtime/Entities/PersistentData/PersistentDataManager.cs

```diff
@@ -0,0 +1,116 @@
+using Anvil.CSharp.Collections;
+using Anvil.Unity.DOTS.Jobs;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Anvil.Unity.DOTS.Entities
+{
+    public static class PersistentDataManager
+    {
+        private static readonly Dictionary<string, AbstractPersistentData> PERSISTENT_DATA_LOOKUP = new Dictionary<string, AbstractPersistentData>();
+        
+        [RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.SubsystemRegistration)]
+        private static void Init()
+        {
+            Dispose();
+        }
+
+        public static void Dispose()
+        {
+            PERSISTENT_DATA_LOOKUP.DisposeAllValuesAndClear();
+        }
+
+        public static void CreatePersistentData<T>(string id,
```

Comment:
ID's as strings... hmmmn. 

With Entities, it's based on `Types` and the amalgamation of them into `Archetypes` plus/or `Entity` as a key for CDFE/DynamicBuffer access.

We could use Types as well, but then it prevents us from having more than one persistent data of the same type. 

For example we have a PathBuffer one for a `UnsafeList<int2>`, I'm pretty sure we'll need `UnsafeList<int2>` for another one that isn't related to PathBuffers. You could get around this by wrapping the `UnsafeList<int2>` into a struct though but that's annoying to create that type just to ensure duplicate inner type usage.

So strings as ID's made the most sense for now. Thoughts?